layouts/partials/featured-image.html

```diff
@@ -9,6 +11,8 @@
     <figcaption class="mt-2 mt-md-0"><small class="text-muted">{{ . | html }}</small></figcaption>
   {{ end }}
   </figure>
+{{ else if strings.Contains $title $falco_weekly_prefix }}
```

Comment:
Since it's a prefix, is it possible to do something like `hasPrefix`?

Reply:
Done!